doc/qbk/01_overview.qbk

```diff
@@ -0,0 +1,49 @@
+[/
+    Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
```

Comment:
For any files that you've copied content from some now-deleted file, also copy the Copyright from those files.

Reply:
Any new content should be `Copyright 2021 Alan de Freitas`